### PR TITLE
Add list of anonymized data points to documentation

### PR DESCRIPTION
**Pull request by natiiix**
This PR adds a Markdown document with a list of the majority of anonymized data points (resource fields). This list was compiled by making reverse-searches of the anonymization helper functions, so it is very possible that there are anonymized data points missing from this list. If you are aware of any, please let me know, and I will gladly add them.

There were quite a few trailing spaces at the ends of lines in the README file, which were automatically removed by my formatter, and I feel like there's no good reason to have them there, so I kept it in this PR.

## Categories

- [ ] Bugfix
- [ ] Enhancement
- [ ] Backporting
- [x] Others (CI, Infrastructure, Documentation)

## Sample Archive

N/A

## Documentation

- `docs/anonymized-data.md`

## Unit Tests

N/A

## Privacy

Yes. This is just a documentation update.

## Changelog

N/A

## Breaking Changes

No

## References

Jira Task: https://issues.redhat.com/browse/CCXDEV-7267

### CODE REVIEW

**Comment by tremes** [comment]
@radekvokal please take a look

File: docs/anonymized-data.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Anonymized Data
+
+This is an approximate list of data point (Kubernetes/OpenShift resource fields)
+that are anonymized by the Insights Operator. This list is subject to change and
+may not include all anonymized fields.
```
Comment (radekvokal) [inline docs/anonymized-data.md:5]: Is it worth explaining why this list could be incomplete? The wording used here leaves some open questions

**Comment by tremes** [comment]
Thanks!
/lgtm

**Pull request merged**